packages/grafana-ui/src/components/PanelChrome/HoverWidget.tsx

```diff
@@ -48,10 +48,11 @@ export function HoverWidget({ menu, title, dragClass, children, offset = -32 }:
         onPointerUp={onPointerUp}
         ref={draggableRef}
       >
-        <Icon name="draggabledots" />
+        <Icon name="expand-arrows" className={styles.draggableIcon} />
       </div>
+      {!title && <h6 className={classnames(styles.untitled, styles.draggable, dragClass)}>Untitled</h6>}
```

Comment:
In the codebase we use `classnames` default export as: `cx`, `classNames`, or `classnames` 🙄  Did we agree to use `classnames` in the end? a bit confusing 😢

Reply:
I modified the code to use cx :)